<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestDecoratorClass</class>
 <widget class="QMainWindow" name="TestDecoratorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestDecorator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="cInput">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>input</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cOutput">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>output</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cOutputOrigin">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>outputOrigin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cStd">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>控制台</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cWidget">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>窗口</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TestDecorator.qrc"/>
 </resources>
 <connections/>
</ui>
